_posts/2018-01-18-From-Manual-to-Automatic.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+authors: Hugo Fernandes, José Ferreira, Sofia Gomes, Elaine Navalho, João Ventura
+title: From Manual to Automatic
+---
+
+In the more recent years the interest on test automation has been increased significantly. Some of the reasons behind this growth include a greater need for faster software delivery processes, increased productivity and cost/effort reduction associated to the tests itself. This article will focus on test automation, its advantages and drawbacks and when and how should the transition from manual to automatic be made. 
+The article will also provide some examples of test automation tools, programming languages used for test automation and continuous integration software.
+
+## How vital is test automation in today’s world. 
+
+We live in a digital age. Every day new applications are deployed in productions systems and product features are improved, adjusted and updated with the objective of promoting the same products in an increasingly competitive market.
+The balance between shorter development cycles, more challenging *time to market* times and quality delivery is nowadays put to the test more than ever. Taking this into mind, many software firms have started to adopt automation methodologies to reduce time and money. In software testing the test automation promotes three significant benefits <sup>[1]</sup>: 
+
+- Cumulative coverage of previously developed features, increasing defect detection and reducing bug fixing costs; 
+- Reproducibility by ensuring software stability and reduced market times and costs; 
+- More effective resource management, resulting in increased productivity. 
+
+To better demonstrate the advantages of test automation when compared to manual testing, our team conducted a basic study with five manual and automatic tests on an internal prototype webapp. As provided in the Appendix I of this article, it has been considered the average metrics for the human resources effort costs for implementing both the manual tests and the automation tests. Because it has been assumed that the maintenance costs were equivalent in both approaches, it has been decided that these were not to be included in the study.
```

Comment:
`it has been considered the average metrics for the human resources effort costs for implementing both the manual tests and the automation tests`
I'd replace with
`the average metrics for the human resources effort costs for implementing both the manual tests and the automation tests have been considered`

I'd also change
`Because it has been assumed that the maintenance costs were equivalent in both approaches, it has been decided that these were not to be included in the study.`
to
`Since maintenance costs were assumed to be equivalent in both approaches, it has been decided that these would not be included in the study.`

Reply:
Changed